app/views/users/new.html.erb

```diff
@@ -1,28 +1,25 @@
 <div id='clearance' class='sign-up'>
-  <h2><%= t('.title') %></h2>
+  <h2>Sign up Now!!</h2>
 
   <%= form_for @user do |form| %>
-    <div class='text-field'>
-      <%= form.label :name %>
-      <%= form.text_field :name, :type => 'name' %>
+
+    <div class='text-field info'>
+      <%= form.text_field :name, :type => 'name', placeholder: 'Name..' %>
```

Comment:
I'm a bit of a stickler for detail — but the rest of the fields have `...` and this one has `..` — probably best to stick to 3! There are a few instances of having just 2 throughout I believe?